samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -179,6 +179,9 @@ AppController.prototype.hangup_ = function() {
   this.transitionToDone_();
 
   this.call_.hangup();
+
+  trace('Reloading main page.');
```

Comment:
I tried this, but I think we will want to retain the end-of-call page, perhaps to display a call rating widget or some such. But let's add a link in the status area to take you back to the landing page, e.g. [Click here] to rejoin, or [start a new room].

Also, can we make the rejoin-room link immediately put you back into the room (i.e. act as if you clicked Join from the landing page)?